features/facebookImport/rollup.config.js

```diff
@@ -60,6 +61,11 @@ export default (commandLineArgs) => {
                     },
                 ],
             }),
+            genPodjs.loadManifest({
```

Comment:
I guess this generates pod.js, since it's been substituted above, but the optics about that are not totally clear. Also, the fact that you need to call `loadManifest` explicitly. Couldn't you simply call `genPodjs`? That way it would be clear that youre `gen`erating pod.js with whatever arguments are needed for that.